### DISCUSSION TITLE
Deleted README.md accidentally and now get error

### DISCUSSION THREAD

**Discussion by EddiesTech** [upptime/upptime#248] [Q&A] [answered]
I accidently deleted the README.md file and now I get the error:
`ENOENT: no such file or directory, open 'README.md'`
Any ideas on how I could fix this? :)
I tried adding a new README.md file, but I get the same error.

**Comment by EddiesTech** [comment, marked as answer]
Fixed by adding content to the README.md file. It no longer edits the README, but I can live with that :)

**Reply by thomasmerz** [reply]
Why don't you do a `git checkout README.md` from a earlier ref?
